hyperlink/test/common.py

```diff
@@ -0,0 +1,70 @@
+
+
+from unittest import TestCase
+
+
+class HyperlinkTestCase(TestCase):
```

Comment:
I mean, I guess the question is only: do we believe 2.6 support is necessary? I suspect the answer to that question boils down to "do we support a version of Twisted that supports 2.6", but if you think 2.6 support is minimal effort then I'm not going to push you to drop it. :wink: